src/docs/content/builders/dropdown-menu.md

```diff
@@ -84,6 +88,40 @@ occurring.
 </div>
 ```
 
+### Modal Behavior
+
+To give the dropdown menu complete modal behavior, where an overlay is rendered behind the menu to
+prevent interaction with the rest of the page while open, use the `overlay` builder element.
```

Comment:
I don't think we should requuire an `overlay` to block interaction outside. A `modal` prop would make more sense. 

Also, I've never seen an overlay for menu components like ours, is it really needed?

Reply:
A `modal` prop would make sense if we were a component library because then we could conditionally render an overlay (as is the case with Radix and others). Unless we want to inject a node into the DOM and take the ability to control the overlay from the user I'm not sure it makes more sense.

A `modal` prop would only make sense if we removed `closeOnOutsideClick`, `preventScroll`, and `closeOnEscape` in favor of a single prop. Not sure if users would like this since it removes flexibility.